virus/Thogotovirus_aidenensis.md

```diff
@@ -0,0 +1,148 @@
+
+## Results
+
+### Ecology subsection A...
+
+sOTU u507489, designated Thogotovirus aidenense, was identified in seven distinct sequencing runs encompassed
+within four unique bioprojects, as revealed by Serratus. The run I chose to assemble was
+SRR16693885, which had the highest coverage of the virus. This run was part of the bioproject PRJNA764088, which
+is linked to a 2023 study at Wageningen University in the Netherlands (Steenbrugge et al, 2023).
+The authors were studying transcription factor binding motifs upstream of 'effector' genes, which
+are responsible for conferring virulence to the plant parasitic nematodes Globoerda rostochiensis
+and Globodera pallida. For this study, they constructed new reference genomes for both species, using
+strains for both that were uncharacteristically hypovirulent. Thogotovirus aidenense was not identified in the
+G. pallida strain, but was found in G. rostochiensis. The authors did not comment on the virus, and
+were likely unaware of its presence. 
+
+The specific line of G. rostochiensis used in the study , 'Gr-line 19,' traces back to an earlier
+study at Wageningen (Janssen et al, 1990). Here, the authors produced 63 F3 lines as a result of
+inbreeding a wild avirulent population. Gr-line 19 did not produce any cysts on potato cultivar Saturna
+and was thus considered avirulent. One of the other bioprojects (PRJNA471544) used the same strain of 
+G. rostochiensis (Sabeh et al, 2019). Notably, they also sequenced several other species of Globodera, 
+but the virus was not detected in any of these samples. 
+
+As for the other two, it's unclear what strains were used. One of these bioprojects (PRJEB2896) is unpublished,
+but notably contains a run from a different species, G. pallida, where the virus was identified. The coverage is
+comparatively low, however, and it remains to be seen if the virus is capable of infecting multiple species.
+The other does not identify the strain, only the pathotype (PRJEB12075), and that the nematodes were obtained from 
+the James Hutton Institute in Scotland (Eves-van den Akker et al, 2016). Interestingly, the authors note their 
+transcript selection strategy used oligo-dT beads, which suggests the virus has polyadenylated transcripts.
+
+### Virus Genome {Q3}
+
+![Genome Map](./img/thogoAiden/genomemap.png)
+
+![RNA Secondary Structure](./img/thogoAiden/rnass.png)
+
+### Other (bonus) sections
+
+{Q4: Create sub-sections for any bonus or other cool findings you've made
+related to your virus}
+
+### Structural Analysis
+
+![PB1 Motifs](./img/thogoAiden/pb1_motifs.png)
+
+While the RdRp is found on a single segment, the polymerase holoenzyme in Thogotoviruses is composed of 
+three separate subunits: PB1 (RdRp-containing, picture above), PB2, and PA.
+
+### Phylogenetic Analysis
+
+![PA Phylogeny](./img/thogoAiden/PAPhylo.png)
+
+![PB1 Phylogeny](./img/thogoAiden/PB1Phylo.png)
+
+![PB2 Phylogeny](./img/thogoAiden/PB2Phylo.png)
```

Comment:
For "multi-panel" figures you can use one Figure Legend for multiple panels/images.